src/main/java/com/rocketden/main/service/ProblemService.java

```diff
@@ -241,7 +244,7 @@ public ProblemTestCaseDto createTestCase(String problemId, CreateTestCaseRequest
 
     // Check to make sure test case inputs and outputs are Gson-parsable
     protected void validateInputsGsonParseable(String input, List<ProblemInputDto> types) {
-        if (input == null) {
+        if (input == null || input.isEmpty()) {
```

Comment:
Good catch - I ended up removing this check. However, it also turns out that GSON can't parse empty strings (it just returns null, which actually is a new bug that I just now fixed/added a test case for), so that might be another issue with using empty strings

On the bright side, while users won't be able to specify ` `, they can still just do `""` (not `\"\"`) to represent an empty string instead - lmk what your thoughts are on all this and what your preferences are moving forward on this

Reply:
That's a good point -- how should users treat this frontend setting then? Should they type in `""` for empty strings (meaning we may be able to add back the `input.isEmpty()` check?), or should they be typing in ``, and we handle the empty issue on the backend as a special check case?

Reply:
I might lean towards the former (less chance for unintentional mistakes when inputs are left blank) but I don't feel too strongly about this so I'm open to implementing something like the second if you prefer that more

Reply:
I agree with you, let's do the former -- you can merge this PR now! 👍